drivers/generic/generic.go

```diff
@@ -83,33 +81,30 @@ func (d *Driver) GetSSHUsername() string {
 func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 	d.IPAddress = flags.String("generic-ip-address")
 	d.SSHUser = flags.String("generic-ssh-user")
-	d.SSHKey = flags.String("generic-ssh-key")
+	d.sourceSSHKey = flags.String("generic-ssh-key")
 	d.SSHPort = flags.Int("generic-ssh-port")
 
 	if d.IPAddress == "" {
-		return fmt.Errorf("generic driver requires the --generic-ip-address option")
+		return errors.New("generic driver requires the --generic-ip-address option")
```

Comment:
If you're going to be changing these to `errors.New` anyway, looks like a good candidate for a custom `ErrRequiredFlagMissing` type.